src/library/Uno.Material/Styles/Application/v2/Typography.xaml

```diff
@@ -15,7 +15,8 @@
 			<!-- DISPLAY -->
 
 			<!--  Display Large  -->
-			<x:String x:Key="DisplayLargeFontFamily">MaterialRegularFontFamily</x:String>
+			<StaticResource x:Key="DisplayLargeFontFamily"
+							ResourceKey="MaterialRegularFontFamily" />
```

Comment:
```suggestion
			<StaticResource x:Key="DisplayLargeFontFamily" ResourceKey="MaterialRegularFontFamily" />
```

Reply:
@vatsashah45 let's apply this to every `<StaticResource ... />` line that you added. It's ok for `StaticResource` elements to have both attributes on the same line